styles/core.scss

```diff
@@ -97,3 +97,33 @@ SPDX-License-Identifier: Apache-2.0 */
   flex-direction: column;
   flex: 1;
 }
+
+.side-interface {
+  height: 100vh;
+  display: flex;
+  flex-flow: column nowrap;
```

Comment:
`nowrap` is the default value. We can just use `flex-direction: column` here as it is the only changing property.